.ci/pipeline/docs.yml

```
@@ -66,16 +66,23 @@ jobs:
   - script: |
       export PREFIX=$(dirname $(dirname $(which python)))
       export DALROOT=$PREFIX
+      export LD_LIBRARY_PATH=$(dirname $(dirname $(which python)))/lib:$LD_LIBRARY_PATH
       ./conda-recipe/build.sh
+      cp ${DALROOT}/lib/python3.11/site-packages/daal4py/_daal4py.cpython*.so daal4py/
```

```suggestion
      NO_DPC=1
      python setup.py build_ext --inplace
```
If I understand correctly, I think you want the _daal4py.so just so you can import the modules of the repo in place.  You can just build the extension, which will suffice. Do you also need to import ```onedal``` for the docs?

---

I think `NO_DPC` should not be necessary if the compiler doesn't support `NO_DPC`, but `NO_DIST=1` might be required.

---

Note that `build_ext` will only build `daal4py`. For `sklearnex` you need `build`.

---

Hi Ethan! Thx for this suggestion, this works and I can remove setting LD_LIBRARY_PATH and cp the build file back. I do need the NO_DIST=1 otherwise it will complain MPIROOT not being set. But I still cannot remove setting LD_LIBRARY_PATH in build doc stage it will gives the module not found warning if not set.
Best,
Yue

---

@yuejiaointel A call to `python setup.py build` also builds the module in-place. One is for `daal4py` and the other for `onedal`+`sklearnex`. They are built using different frameworks which follow different setup commands.

---

Amendment: looks like `LD_LIBRARY_PATH` is necessary for TBB regardless of `PYTHONPATH`. (experiments here: https://github.com/uxlfoundation/scikit-learn-intelex/pull/2269)